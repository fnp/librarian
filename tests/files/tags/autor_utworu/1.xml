<autor_utworu>Hans Christian Andersen</autor_utworu>
