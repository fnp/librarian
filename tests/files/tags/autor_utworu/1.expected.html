<div id="book-text"><h1><span class="wl author" id="i1">Hans Christian Andersen</span></h1></div>
